env0/test_helpers.go

```diff
@@ -2,8 +2,9 @@ package env0
 
 import (
 	"fmt"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"regexp"
+
```

Comment:
Why empty line

Reply:
It come up when I am save the file, tried to fix it but then failed in fmt test job

Reply:
@samuel-br  👍